doc/2/guides/getting-started/1-run-kuzzle/index.md

```diff
@@ -92,7 +98,7 @@ The Admin Console is a [Single Page Application](https://en.wikipedia.org/wiki/S
 No data related to your connection to Kuzzle will pass through our servers.
 :::
 
-First, we need to setup a new connection to a Kuzzle application. Open the [Admin Console](http://console.kuzzle.io) in your browser and then fill the form as follow:
+First, we need to setup a new connection to a Kuzzle application. Open the [Admin Console](http://next-console.kuzzle.io) in your browser and then fill the form as follow:
```

Comment:
we will have to change the link to console.kuzzle.io once the v4 is released...

Reply:
Yes I know but otherwise it's confusing for people who I ask to test the Getting started :see_no_evil: